_includes/charts/calico/crds/kdd/crd.projectcalico.org_bgpconfigurations.yaml

```diff
@@ -54,7 +54,7 @@ spec:
                   cluster IPs are allocated. If specified, Calico will advertise these
                   blocks, as well as any cluster IPs within them.
                 items:
-                  description: ServiceClusterIPBlock represents a single whitelisted
+                  description: ServiceClusterIPBlock represents a single allow-listed
```

Comment:
This doesn't need to have allow-listed in it, if we remove the word the same definition holds true :-).

Reply:
```suggestion
                  description: ServiceClusterIPBlock represents a single
```

Reply:
For this one specifically as we're mentioning a single block it's self-documenting to me to not have the word allowed at all in there. I don't believe this will be user-hostile to not have the phrase present (obviously we're all human though 😉), just throwing the idea out there.